.github/workflows/stale-pull-requests.yml

```
@@ -1,37 +1,48 @@
-name: Check stale pull requests
+name: Check stale issues and pull requests
 
 on:
   workflow_dispatch:
   schedule:
     - cron: '0 12 * * *'
 
 permissions:
-  issues: read
+  issues: write
   pull-requests: write
 
 env:
-  BEFORE_STALE: 14
-  BEFORE_CLOSE: 7
+  BEFORE_ISSUE_STALE: 180
+  BEFORE_ISSUE_CLOSE: 0 #FIXME: change to 14 days
+  BEFORE_PR_STALE: 14
+  BEFORE_PR_CLOSE: 7
 
 jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/stale@v6
         with:
-          debug-only: false
-          # disable issues
-          days-before-issue-stale: -1
-          days-before-issue-close: -1
+          debug-only: true
+          days-before-issue-stale: ${{ env.BEFORE_ISSUE_STALE }}
+          days-before-issue-close: ${{ env.BEFORE_ISSUE_CLOSE }}
+          stale-issue-message: |
+            Hi everyone! This issue has been marked as stale due to inactivity for the last ${{ env.BEFORE_ISSUE_STALE }} days.
+            It will be automatically closed in ${{ env.BEFORE_ISSUE_CLOSE }} days.
+            If you think this issue is still relevant in the latest Solidity version, please feel free to open a discussion in our [forum](https://forum.soliditylang.org/).
+          close-issue-message: |
+            Hi everyone! This issue has been closed due to inactivity.
+            Feel free to reopen if you have something to [contribute](https://docs.soliditylang.org/en/latest/contributing.html).
```

Well, I meant the forum part to be added when we close because that's when it's relevant. Not when it becomes stale.

Here's how I'd word it:
```suggestion
            If you think this issue is still relevant in the latest Solidity version and you have something to [contribute](https://docs.soliditylang.org/en/latest/contributing.html), feel free to reopen.
            However, unless the issue is a concrete proposal that can be implemented, we recommend starting a language discussion on the [forum](https://forum.soliditylang.org) instead.
```

---

Oh, sorry, I misunderstood it.